### DISCUSSION TITLE
How about making copy function (Edit Menu) work?

### DISCUSSION THREAD

**Discussion by dwgillies-bluescape** [MuhammedKalkan/OpenLens#136] [Ideas]
OpenLens 6.4.6
Extension API 6.4.6
Electron 19.1.9
Chrome 102.5005.67.167

In my experience, when I shell onto an AWS instance or shell into an AWS pod, cut & past don't work.

Command-C   - does nothing
Edit Menu > Copy - does nothing
Command-V   - sometimes works, sometimes doesn't

I am on a 2019 Intel Mac running 11.7.2 Big Sur
cut & paste works everywhere else.

**Comment by FaraSys** [comment]
I have the same problem.
OS: Linux Debian 11
